Makefile

```diff
@@ -191,14 +197,23 @@ install: ## install the linux binaries
 
 run: build ## run the docker daemon in a container
 	$(DOCKER_RUN_DOCKER) sh -c "KEEPBUNDLE=1 hack/make.sh install-binary run"
-
+ 
+.PHONY: build_shell
 ifeq ($(BIND_DIR), .)
-shell: DOCKER_BUILD_ARGS += --target=dev
+build_shell: shell_target := --target=dev
 else
-shell: DOCKER_BUILD_ARGS += --target=final
+build_shell: shell_target := --target=final
+endif 
```

Comment:
nit: extra space at EOL

Reply:
😟 That's what I get for changing editors
Think I got them all.